client/src/components/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./button.css";
+
+const Button = ({ children, ...allAttributes }) => (
```

Comment:
I'd prefer this is written on a single line, but that's not a **red** requested change.